### PR TITLE
monitoring: use pointer for typed value

### PR DESCRIPTION
**Pull request by broady**
The API client was changed:
https://code-review.googlesource.com/5259

### CODE REVIEW

**Comment by broady** [comment]
@waprin @mcgreevy

**Comment by waprin** [comment]
lgtm

File: monitoring/custommetric/custommetric.go [hunk 1]
```diff
@@ -88,7 +89,7 @@ func writeTimeSeriesValue(s *monitoring.Service, projectID, metricType string) e
 					EndTime:   now,
 				},
 				Value: &monitoring.TypedValue{
-					Int64Value: rand.Int63n(10),
+					Int64Value: &randVal,
```
Comment (mcgreevy) [inline monitoring/custommetric/custommetric.go]: Alternatively, this could be:

Int64Value: googleapi.Int64(rand.Int63n(10))

Comment (broady) [inline monitoring/custommetric/custommetric.go]: Yeah, I figured. I didn't want to import another package. But I guess that's why it's there.

**Pull request merged**